WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -195,6 +195,13 @@ final class ProductFormViewController<ViewModel: ProductFormViewModelProtocol>:
             }
         }
 
+        if viewModel.canDeleteProduct() {
+            actionSheet.addDefaultActionWithTitle(ActionSheetStrings.delete) { [weak self] _ in
```

Comment:
nit: maybe the "Delete" action could be destructive?
```suggestion
            actionSheet.addDestructiveActionWithTitle(ActionSheetStrings.delete) { [weak self] _ in
```

Reply:
I'm not totally 100% sure about it, since it will differ from the original design proposed by @Garance91540, visible [here](https://github.com/woocommerce/woocommerce-ios/issues/2853).

This is the result of using the `Destructive` option.
<img src="https://user-images.githubusercontent.com/495617/94150198-209ed700-fe79-11ea-83d8-62c9eaa532af.png" width=300 />

Reply:
We usually use the destructive for deleting/removing something from the site, like deleting a tracking number from order details or "Discard changes" in the product form discard changes action sheet. Maybe "Product Settings" can be moved up so that "Delete" is at the bottom @Garance91540 ?